tools/kochdocs.nim

```diff
@@ -105,7 +105,7 @@ doc/manual/var_t_return.rst
   doc = """
 lib/system.nim
 lib/system/nimscript.nim
-lib/pure/ospaths.nim
+lib/depreceated/pure/ospaths.nim
```

Comment:
typo: deprecated

Reply:
Ah appreciate it. But I think moving the file caused issues with the tests. There was an error for a file not found.